src/main.tsx

```diff
@@ -1,12 +1,13 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App.tsx'
 import { ToastContainer } from 'react-toastify';
+import { createRoot } from 'react-dom/client';
+import { StrictMode } from 'react';
+
+import { App } from './App.tsx';
 import 'react-toastify/dist/ReactToastify.css';
 
-ReactDOM.createRoot(document.getElementById('root')!).render(
-  <React.StrictMode>
+createRoot(document.getElementById('root')!).render(
+  <StrictMode>
     <App />
     <ToastContainer position="bottom-left" autoClose={2000} />
-  </React.StrictMode>,
+  </StrictMode>,
```

Comment:
Also for this, can understand?

Reply:
It's refactoring, just for reduce code size and make it more readable